### ISSUE TITLE
[Vanilla Bug Report] v2.0, strange alpha/opacity on full LOD cables found on Broker Bridge

### ISSUE THREAD

**Issue by Abbanon** [valentyn-l/GTAIV.EFLC.Various.Fixes#60]
The entirety of Broker Bridge has a bunch of support cables, all of which feature strangely transparent alpha/opacity at close range, as though they're made of the same material as traditional power lines:
![Screenshot_8](https://github.com/user-attachments/assets/cc284488-f3f0-4b7a-bcbb-2e7ba352d865)
![Screenshot_6](https://github.com/user-attachments/assets/2e84c782-1be6-49b6-8399-95d96c23d41c)

The cables actually appear more solid when far away, and only become transparent when you get close enough:
![Screenshot_9](https://github.com/user-attachments/assets/78c5b6fe-e43c-4e62-ad50-745dddaed376)
![Screenshot_10](https://github.com/user-attachments/assets/824f48c6-ca60-44f2-874d-5b20ff8e96bd)

**Comment by RecklessGlue540** [comment]
Wires which use gta_wire shader will have a quite strong blur effect applied to avoid aliasing, it's intended behavior although a very big chunk of wire models do not use that shader and instead they use gta_default which makes them opaque, and if enabled in .ides for any wire model using gta_default, they can cast sun shadows as well. 

They will not be able to cast shadows with gta_wire however, not unless the int 1 in gta_wire.sps is changed to int 0, which brings back their opaque appearance and aliasing even though the shader is correct.
Thing is if you force them to cast shadows in the .ide flags, **but without the int tweak in the .sps**, so forcing them to cast shadows while being blurred **may** end up breaking them completely, as explained by Parallellines [here](https://github.com/ThirteenAG/GTAIV.EFLC.FusionFix/issues/581)
Whether or not it's the case with wires as well, for example that broken lighting, since the wire shader is forward rendering instead of deferred, I have no idea. 
Anyone may use the list I provided there to check any wire model with gta_wire + shadow casting flag in .ide for said model + no .sps tweak, int 1 basically, and seeing if the shadow casting makes said wire model go broken while blurred with the correct shader.
Since then, if it works no issues on wires, we could have shadow casting for every wire in Various Fixes as an optional eventually, that includes just the required shader preset that makes them opaque, with the trade-off of having them extremely aliased (Some like me would rather see their shadows than their aliasing for example)

Once again, it's a pretty big issue to mess with, as there are hundreds of wire models in the game and like I said a big chunk would need to be given the proper shader first, since the blurring was always intended and any wires which use something else means they were overlooked.

**Comment by RecklessGlue540** [comment]
Yeah hopefully, but it's similar in scope to the one with the overly-reflective buildings and terrains, so it's probably not going to be so soon.
